src/bin/20.rs

```diff
@@ -0,0 +1,39 @@
+// n! means n × (n − 1) × ... × 3 × 2 × 1
+//
+// For example, 10! = 10 × 9 × ... × 3 × 2 × 1 = 3628800,
+// and the sum of the digits in the number 10! is 3 + 6 + 2 + 8 + 8 + 0 + 0 = 27.
+//
+// Find the sum of the digits in the number 100!
+
+extern crate num_bigint;
+extern crate num_traits;
+
+use num_bigint::BigUint;
+use num_traits::FromPrimitive;
```

Comment:
@michaelkamau  `FromPrimitive` is unused. Maybe remove?

Reply:
I've updated the CI to deny warnings. Hopefully it will make it easier to catch this in future.